### PR TITLE
Qf 2661 move certain exceptions to subscription

### PR DESCRIPTION
**Pull request by faebebin**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2661 move exceptions to subscriptions module](https://app.clickup.com/t/2192114/QF-2661)

File: docker-app/qfieldcloud/core/exceptions.py [hunk 1]
```diff
@@ -169,6 +162,7 @@ class ProjectAlreadyExistsError(QFieldCloudException):
     status_code = status.HTTP_400_BAD_REQUEST
 
 
+@deprecated("moved to subscription")
```
Comment (suricactus) [inline docker-app/qfieldcloud/core/exceptions.py:165]: Do we really need to deprecate them, can't we just delete them?

File: docker-app/qfieldcloud/subscription/exceptions.py [hunk 2]
```diff
@@ -1,6 +1,56 @@
...
+
+class SubscriptionException(QFieldCloudException):
     ...
 
 
 class NotPremiumPlanException(SubscriptionException):
     ...
+
+
+class ReachedMaxOrganizationMembersError(QFieldCloudException):
```
Comment (suricactus) [inline docker-app/qfieldcloud/subscription/exceptions.py:13]: Shouldn't all of these onwards extend `SubscriptionException`?

File: docker-app/qfieldcloud/core/exceptions.py (hunk 1 above)
Comment (faebebin) [reply docker-app/qfieldcloud/core/exceptions.py:165]: oh sure, only keep  ReachedMaxOrganizationMembersError

**Review by suricactus — APPROVED** [review]
Thanks!

**Pull request merged**